tests/view_tests/tests/test_debug.py

```diff
@@ -596,72 +580,89 @@ class ExceptionReporterFilterTests(TestCase, ExceptionReportTestMixin):
     Ensure that sensitive information can be filtered out of error reports.
     Refs #14614.
     """
-    rf = RequestFactory()
+    client_class = SensitiveTestClient
+
+    def setUp(self):
+        logger = logging.getLogger('django.request')
+        log_handler = logger.handlers[0]
+        # Disable filter for DEBUG=True
+        filters = log_handler.filters
+        self.addCleanup(lambda: setattr(log_handler, 'filters', filters))
```

Comment:
I'd move this monkeypatch stuff to `tearDown`. `addCleanup` is suitable for directory, file or another resource cleanups.